pegasus/sites.v3/code.org/public/css/mobile_header.css

```diff
@@ -0,0 +1,165 @@
+#pageheader-wrapper {
+  z-index: 5;
+}
+
+#nav-hamburger {
+  cursor: pointer;
+  padding: 10px 35px 16px 0px;
+  height: 20px;
+}
+
+#nav-hamburger span, #nav-hamburger span:before, #nav-hamburger span:after {
+  cursor: pointer;
+  border-radius: 1px;
+  height: 3px;
+  width: 25px;
+  border-radius: 2px;
+  background: white;
+  position: absolute;
+  display: block;
+  content: '';
+  margin-top: 2px;
+  margin-bottom: 2px;
+}
+
+#nav-hamburger span:before {
+  top: -10px;
+}
+
+#nav-hamburger span:after {
+  bottom: -10px;
+}
+
+#nav-hamburger span, #nav-hamburger span:before, #nav-hamburger span:after {
+  transition: all 500ms ease-in-out;
+}
+
+#nav-hamburger.active span {
+  background-color: transparent;
+}
+
+#nav-hamburger.active span:before, #nav-hamburger.active span:after {
+  top: 0;
+}
+
+#nav-hamburger.active span:before {
+  transform: rotate(45deg);
+}
+
+#nav-hamburger.active span:after {
+  transform: rotate(-45deg);
+}
+
+#nav {
+  display: inline-block;
+  text-align: left;
+  float: right;
+  width: 50%;
+}
+
+#nav ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 250px;
+  right: 0;
+}
+
+#nav ul li {
+  font-size: 18px;
+  color: white;
+  display: inline-block;
+  float: left;
+  padding: 0.25em 0em;
+  margin-right: 1em;
+  text-indent: 0;
+  list-style-type: none;
+  max-width: 150px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+#nav ul li:before {
+  content: none;
+}
+
+#nav ul li:last-child {
+  margin-right: 0;
+}
+
+#nav ul li a, #about-more {
+  font-family: 'Gotham 4r', sans-serif;
+  font-size: 18px;
+  color: white;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+#about-more {
+  width: 90px;
+  cursor: pointer;
+}
+
+#nav ul li a:hover, #about-more:hover {
+  border-bottom: solid 2px rgb(255, 138, 0);
+}
+
+#nav-hamburger {
+  vertical-align: middle;
+  text-decoration: none;
+  display: none;
+  float: right;
+  margin-top: 8px;
+}
+
+.about-nav li a {
+  margin-left: 20px;
+}
+
+@media (max-width: 992px) {
+  #pageheader-wrapper {
+    padding-top: 5px;
+  }
+
+  #nav{
+    position: relative;
+  }
+
+  #nav ul.hide-responsive-menu, .hide-about {
+    display: none;
+  }
+
+  #nav ul {
+    position: absolute;
+    top: 4em;
+    background-color: rgba(0, 178, 192, .9);
+    z-index: 99;
+  }
+
+  #nav ul:after {   /* This is the arrow in the responsive menu */
```

Comment:
Same nit about comments (check your whole PR).

* // Have a space between the comment marks
* Start capitalized and are punctuated like a sentence.